### ISSUE TITLE
[Flaky Test] kind-{ipv6}-master-parallel (ci-kubernetes-kind-{ipv6}-e2e-parallel)

### ISSUE THREAD

**Issue by hasheddan** [kubernetes/kubernetes#91548]
**Which jobs are flaking**:

`kind-master-parallel (ci-kubernetes-kind-e2e-parallel)`
`kind-ipv6-master-parallel (ci-kubernetes-kind-e2e-parallel)`

**Which test(s) are flaking**:

Pretty noisy across the board causing timeouts, but the following appear to be especially noisy:
`[sig-storage] CSI mock volume CSI Volume expansion should expand volume by restarting pod if attach=off, nodeExpansion=on`
`[sig-storage] Projected secret should be consumable from pods in volume [NodeConformance] [Conformance]`

**Testgrid link**:

https://testgrid.k8s.io/sig-release-master-blocking#kind-master-parallel
https://testgrid.k8s.io/sig-release-master-blocking#kind-ipv6-master-parallel

**Reason for failure**:

`Timed out waiting for condition` across multiple tests

**Anything else we need to know**:
/cc @kubernetes/ci-signal
/priority critical-urgent
/milestone v1.19
/sig testing
/assign @VineethReddy02 
/cc @BenTheElder

**Comment by BenTheElder** [comment]
the release blocking tests were moved to wg-k8s-infra infrastructure.
the node pool is puny compared to the old prow, and I'm seeing nodes at 100% CPU usage...

cc @spiffxp

**Comment by spiffxp** [comment]
FWIW I scaled the pool around 3:30pm PT today (ref: https://github.com/kubernetes/k8s.io/pull/930#issuecomment-636540010)

**Comment by dims** [comment]
thanks @spiffxp

**Comment by hasheddan** [comment]
Thank you @spiffxp! Will continue to monitor and close if we stay green

**Comment by BenTheElder** [comment]
Still flakes but greener

**Comment by hasheddan** [comment]
@BenTheElder any updates here? These jobs have seen quite a few flakes recently

**Comment by BenTheElder** [comment]
In presubmit we blocked some test cases that are being worked on (e.g. conntrack related tests have problems on multiple platforms).
If we need we can remove these from the release signal too, but I hope we'll actually get the issues addressed.
I'm also not sure off the top of my head which jobs are getting experimental increased CPU requests besides presubmit. I'm concerned that the current CI host cluster may lack capacity.

**Comment by aojea** [comment]
/assign
it seems that this is the most affected 

https://testgrid.k8s.io/sig-testing-kind#kind-ipv6-master-parallel

**Comment by aojea** [comment]
some errors on the failing job

`Jun 24 20:35:30.272: INFO: exec kind-worker2: stderr:    "5120+0 records in\n5120+0 records out\n20971520 bytes (21 MB, 20 MiB) copied, 0.734787 s, 28.5 MB/s\nlosetup: /tmp/local-driver-cb628f37-301e-45ce-a39c-26c579c377bd/file: failed to set up loop device: No such file or directory\n"`

per slack conversation, Ben has an idea of what can be causing those errors
https://kubernetes.slack.com/archives/C2C40FMNF/p1593036217025000?thread_ts=1593035570.022200&cid=C2C40FMNF

**Comment by BenTheElder** [comment]
various changes have been made and the past 10 runs have passed for both jobs.
for now
/close

**Comment by BenTheElder** [comment]
<img width="566" alt="Screen Shot 2020-06-26 at 2 55 09 PM" src="https://user-images.githubusercontent.com/917931/85904249-0e02fc00-b7bd-11ea-9704-9186720cf987.png">

**Comment by aojea** [comment]
thanks Ben

**Comment by hasheddan** [comment]
@BenTheElder looks like we are seeing quite a few flakes here again. Would you prefer we reopen or is this being tracked elsewhere?

**Comment by BenTheElder** [comment]
In recent history on the non-ipv6 job I see:
- bazel build flake (https://prow.k8s.io/view/gcs/kubernetes-jenkins/logs/ci-kubernetes-kind-e2e-parallel/1277542387874271234) which is just the build, not kind specific
- udp services test flakes (already tracked in another issue)

**Comment by BenTheElder** [comment]
IPv6 has the same build flake, and loop setup issues. 
I'm waiting on approval for https://github.com/kubernetes/k8s.io/pull/987

**Comment by RobertKielty** [comment]
kubernetes/k8s.io#987 is merged
